hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/utils/columnSearch.tsx

```diff
@@ -69,8 +69,16 @@ class ColumnSearch extends React.PureComponent {
     filterIcon: (filtered: boolean) => (
       <Icon type='search' style={{color: filtered ? '#1890ff' : undefined}}/>
     ),
-    onFilter: (value: string, record: any) =>
-      record[dataIndex].toString().toLowerCase().includes(value.toLowerCase()),
+    onFilter: (value: string, record: any) => {
+      if (record[dataIndex] !== undefined || record[dataIndex] !== null) {
+          return typeof (record[dataIndex]) === typeof {}
+            ? record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())) ? true :false
```

Comment:
do I understand it correctly, that in this line there is an if condition based on the value of `record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase()))` and we return `true` if it's true and `false` if it's false? 
```suggestion
            ? record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase()))
```

Reply:
find() in JS will return the first matched element else undefined.
What this statement might be doing is if find() returns a match then true, else if undefined, then false.

But can we write this a but more cleaner @smitajoshi12 ?
Something like `Boolean(record[dataIndex].find(.....))`

Reply:
@devabhishekpal 
Abhishek Completed change in recent commit as per your suggestion.

Reply:
@smitajoshi12 could you please check my [comment](https://github.com/apache/ozone/pull/5213#discussion_r1308441011) above? I think that hasn't been addressed in your latest commit and I think there is an unnecessary if statement there, please correct me if I'm wrong.

Reply:
@dombizita  @devabhishekpal 
Boolean (record[dataIndex].find(item => Object.values(item).toString().toLowerCase().includes(value.toLowerCase())))
added below condition in latest commit.